src/frontend/components/PublicVideoLiveJitsi/index.spec.tsx

```diff
@@ -31,22 +47,87 @@ jest.mock('../../data/appData', () => ({
   },
 }));
 
-const mockExecuteCommand = jest.fn();
-const mockJitsi = jest.fn().mockImplementation(() => ({
-  executeCommand: mockExecuteCommand,
-  addListener: jest.fn(),
+jest.mock('components/StudentLiveWrapper', () => ({
+  LiveVideoWrapper: jest.fn(),
 }));
 
-describe('<PublicVideoLiveJitsi />', () => {
-  it('renders jitsi and the leave button', () => {
-    global.JitsiMeetExternalAPI = mockJitsi;
+jest.mock('utils/conversejs/converse', () => ({
+  initConverse: jest.fn(() => jest.fn()),
+}));
+
+describe('<PublicVideoLiveJitsi /> unit tests', () => {
```

Comment:
IMO this describe block can be removed, the second one does exactly the same thing. You exactly test what you expect.

Reply:
And in this case you can remove the component mock